prov/verbs/src/fi_verbs.c

```diff
@@ -47,6 +47,35 @@ size_t verbs_default_tx_iov_limit 	= 4;
 size_t verbs_default_rx_iov_limit 	= 4;
 size_t verbs_default_inline_size 	= 64;
 
+struct fi_ibv_gl_data fi_ibv_gl_data = {
+	.def_tx_size		= 384,
+	.def_rx_size		= 384,
+	.def_tx_iov_limit	= 4,
+	.def_rx_iov_limit	= 4,
+	.fork_unsafe		= 0,
+	.use_odp		= 1,
+	.cqread_bunch_size	= 8,
+	.iface			= "ib",
+
+	.rdm			= {
+		.buffer_num		= FI_IBV_RDM_TAGGED_DFLT_BUFFER_NUM,
+		.buffer_size		= FI_IBV_RDM_DFLT_BUFFER_SIZE,
+		.rndv_seg_size		= FI_IBV_RDM_SEG_MAXSIZE,
+		.thread_timeout		= FI_IBV_RDM_CM_THREAD_TIMEOUT,
+		.eager_send_opcode	= "IBV_WR_SEND",
+	},
+
+	.dgram			= {
+		.use_name_server	= 0,
+		.name_server_port	= 0, /* TODO */
+		.separate_send_recv_cq	= 0,
+		.device = {
+			.port_number	= 0, /* 0 - means all available ports */
+			.name		= NULL,
```

Comment:
Yes, it just initial value.
I've updated PR where value of dgram's variables are read.